cfme/tests/ansible/test_embedded_ansible_services.py

```diff
@@ -76,6 +78,55 @@ def local_ansible_catalog_item(appliance, ansible_repository):
     cat_item.delete_if_exists()
 
 
+@pytest.fixture(scope="function")
+def dialog_with_catlog_item(appliance, ansible_repository, ansible_catalog):
```

Comment:
typo: `dialog_with_**catalog**_item`

Reply:
okay I will update it. Nice catch by the way.